appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleTracker.kt

```diff
@@ -65,8 +56,6 @@ internal class ExperienceLifecycleTracker(
                 when (it) {
                     is RenderingStep -> {
                         if (it.isFirst) {
-                            // update this value for auto-properties
-                            storage.lastContentShownAt = Date()
```

Comment:
this was out of place, now we can easily update this property when ExperienceStarted gets "pre-processed" by SessionService [here](https://github.com/appcues/appcues-android-sdk/pull/479/files#diff-54273f483b977ea6b839f9f1119798f237f0214e1725c22497a6df34933f324aR102)